### ISSUE TITLE
Update translations

### ISSUE THREAD

**Issue by VadiqueMe** [dougmencken/HeadOverHeels#31]
Some items in menus are in english even when I choose русский language

**Comment by dougmencken** [comment]
Well, it’s not only for russian. And I don’t want to google translate for languages I don’t speak... Patches from humans are welcome

**Comment by VadiqueMe** [comment]
> Well, it’s not only for russian. And I don’t want to google translate for languages I don’t speak... Patches from humans are welcome

ok ладно

**Comment by dougmencken** [comment]
I’m going to add “**(EN)**” before the untranslated string when it comes from the “guaranteed” language strings

**Comment by dougmencken** [comment]
@hendrikbez do you wish to participate in updating the translations ?

**Comment by hendrikbez** [comment]
Yes I will help with update the translation


*Jesus Christ is Lord*

*Not a swear word*

**Comment by dougmencken** [comment]
> Yes I will help with update the translation

Here is the af_ZA.xml.new file where all untranslated strings can be easily found
[af_ZA.xml.zip](https://github.com/dougmencken/HeadOverHeels/files/15252010/af_ZA.xml.zip)

**Comment by hendrikbez** [comment]
Here it is, did translate it for you



*Jesus Christ is Lord*

*Not a swear word*

**Comment by dougmencken** [comment]
> Here it is

Where?

**Comment by hendrikbez** [comment]
I did send it, but let me try again, the internet was off for a few minutes
now.

Let me know if you have received it



*Jesus Christ is Lord*

*Not a swear word*

**Comment by hendrikbez** [comment]
Did you received the file


*Jesus Christ is Lord*

*Not a swear word*

**Comment by dougmencken** [comment]
> Did you received the file

Nope. Looks like mail attachments are filtered out

The best git-friendly way to participate is to use [your branch](https://github.com/hendrikbez/HeadOverHeels), put the translated file in gamedata/text replacing the old af_ZA.xml, then `git commit` it and `git push`, and then initiate the merge request

**Comment by hendrikbez** [comment]
ok, I will try add it in git for you

*Jesus Christ is Lord*

*Not a swear word*

**Comment by dougmencken** [comment]
what’s already updated

* [**English** en_US](https://github.com/dougmencken/HeadOverHeels/blob/master/gamedata/text/en_US.xml)
* [**Français** fr_FR](https://github.com/dougmencken/HeadOverHeels/blob/master/gamedata/text/fr_FR.xml)
* [**Afrikaans** af_ZA](https://github.com/dougmencken/HeadOverHeels/blob/master/gamedata/text/af_ZA.xml)

I’m poking the guys who previously helped with translations
@attuska
@VadiqueMe

**Comment by VadiqueMe** [comment]
how can I get the new ru_RU.xml file like Douglas did it for af_ZA

**Comment by dougmencken** [comment]
> how can I get the new ru_RU.xml file like Douglas did it for af_ZA

just select your language in the options, and the game’s console (std)out will dump you the updated file with all english strings

then you can copy’n‘paste it, and save as "ru_RU.xml.new"

to diff it with english translation (or français, or afrikaans, or any other), use
```
diff -u gamedata/text/en_US.xml ru_RU.xml.new
```

**Comment by VadiqueMe** [comment]
It wasn't immediate, but I found it, thanks

> then you can copy’n‘paste it, and save as "ru_RU.xml.new"
> 
> to diff it with english translation (or français, or afrikaans, or any other), use
> 
> ```
> diff -u gamedata/text/en_US.xml ru_RU.xml.new
> ```

**Comment by VadiqueMe** [comment]
Sorry, I closed it by accident

**Comment by dougmencken** [comment]
@VadiqueMe I merged your translation

what’s already updated

* [**English** en_US](https://github.com/dougmencken/HeadOverHeels/blob/master/gamedata/text/en_US.xml)
* [**Français** fr_FR](https://github.com/dougmencken/HeadOverHeels/blob/master/gamedata/text/fr_FR.xml)
* [**Afrikaans** af_ZA](https://github.com/dougmencken/HeadOverHeels/blob/master/gamedata/text/af_ZA.xml)
* [**Russian** ru_RU](https://github.com/dougmencken/HeadOverHeels/blob/master/gamedata/text/ru_RU.xml)

any more?

**Comment by Linutux1** [comment]
Good morning @dougmencken 

If you want, I can update Spanish translation

**Comment by dougmencken** [comment]
> Good morning @dougmencken
> 
> If you want, I can update Spanish translation

sure, how can I say no 😉
